### DISCUSSION TITLE
2021-07-07 Weekly Meeting 11 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#54] [Weekly Meetings]
Weekly Meeting 11 is happening on Wednesday 2021-07-07 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- [Website brainstorm results and next steps](https://hackmd.io/6LzNhWDCT6ito0PjrpnEpQ)
- [Open Collective](https://opencollective.com/omigroup)
- glTF Extensions

**Comment by mrmetaverse** [comment]
created some automated triggers on our kanbans on our github. Now when you create a new issue (if it's issue worthy) it will automatically be added to the corresponding [project backlog](https://github.com/omigroup/website/projects), as well as the [Master OMI](https://github.com/orgs/omigroup/projects) backlog. 

If you have a task that isn't quite issue-worthy, you can simply create a card directly on the backlog. 

Furthermore, if someone mentions something of issue-worthy value in a github discussion, you can actually create an issue directly from that discussion by clicking the eplipses on the top-right of [the discussion](https://github.com/omigroup/website/discussions/4)
